donkeycarManagerFront/src/components/dashboard.vue

```diff
@@ -383,8 +384,13 @@ export default {
     async resumeJobs (player) {
       await srv.resumeJobs(player)
     },
-    async pauseJobs (player) {
-      await srv.pauseJobs(player)
+    async reload (player) {
+      this.reload = true
+      await srv.addJobs(player.player_id)
```

Comment:
You aren't sure the job will be assigned to the same exact car. As the player already have a the remote controller in his hands it would be better to ensure the job is assigned to the same car.

To do that, add an optional argument to `srv.addJobs` with the worker ID, you can send it in the POST request see the swagger for that.
We will need to call your `reload` function with a job a not a player to have access to the worker_id on the current job.